addons/api/tests/unit/utils/mql-query-test.js

```diff
@@ -0,0 +1,208 @@
+import {
+  generateMQLSearchExpression,
+  generateMQLFilterExpression,
+  generateMQLExpression,
+} from 'api/utils/mql-query';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | mql-query', function () {
+  module('generateMQLSearchExpression tests', function () {
```

Comment:
I started using nested modules 🤷, I like the organization more

Reply:
I love it. @calcaide has been looking into this so he may have some input. However, I like the clear use of it in a pretty simple test file.

Reply:
this new structure makes it easier to read tests (esp for unit tests)